lib/components/app/popup.tsx

```diff
@@ -20,21 +25,23 @@ const isMobile = coreUtils.ui.isMobile()
 const PopupWrapper = ({ content, hideModal }: Props): JSX.Element | null => {
   const intl = useIntl()
 
-  const { appendLocale, id, url } = content || {}
+  const { appendLocale, id, iframe, url } = content || {}
   const shown = !!url
 
+  const useIframe = !isMobile && iframe
+
   // appendLocale is true by default, so undefined is true
   const compiledUrl = `${url}${
     appendLocale !== false ? intl.defaultLocale : ''
   }`
 
   useEffect(() => {
-    if (isMobile && shown) {
+    if (!useIframe && shown) {
       window.open(compiledUrl, '_blank')
```

Comment:
As implemented, the button works only once, and clicking it again does nothing. External links in new tabs should be implemented as `<a target="_blank" ...>` (react-bootstrap buttons support that).

Reply:
We can't use an external anchor because not all of our targets are anchor tags and that would be a lot of needley code. I've fixed this problem inside the `Popup` component

Reply:
We'll leave that work for another PR, probably when dealing with a11y.